content/pt-br/docs/tasks/configmap-secret/managing-secret-using-kustomize.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Gerenciando Secret usando Kustomize
+content_type: task
+weight: 30
+description: Criando objetos Secret usando o arquivo kustomization.yaml
+---
+
+<!-- overview -->
+
+Desde o Kubernetes v1.14, o `kubectl` provê suporte para [gerenciamento de objetos usando Kustomize](/docs/tasks/manage-kubernetes-objects/kustomization/).
+O Kustomize provê geradores de recursos para criar Secrets e ConfigMaps. 
+Os geradores Kustomize devem ser especificados em um arquivo `kustomization.yaml` dentro
+de um diretório. Depois de gerar o Secret, você pode criar o Secret na API server com `kubectl apply`.
```

Comment:
maybe we can have some translation not pointing to the API Server (it became confusing here), but something like "você pode criar o Secret com" and skip the API Server